src/components/orgunits/OrgUnitTree.style.js

```diff
@@ -0,0 +1,44 @@
+// TODO: The d2-ui component should use classes, not styles!
+// Copied from https://github.com/dhis2/d2-ui/blob/master/packages/org-unit-dialog/src/styles/OrgUnitDialog.style.js
```

Comment:
I don't love the idea of copying this file, it will get out of sync.  Is it an exact copy?  Could we instead `import orgUnitStyles from '@dhis2/d2-ui-org-unit-dialog/styles/OrgUnitDialog.style'` ?  I know that adds a dependency, though, so it might not be a great solution.  I imagine it would take a lot of work to actually use the `d2-ui` component?

Reply:
Yes, it will be a lot of work to adapt the d2-ui OrgUnitDialog component as we need different setups for different layer types. 

I agree that copying the styles is not ideal, but doing it for now. When we move away from Material UI for these componets, we need to get a better solution. 

As the Maps app bundle is already quite huge, I would prefer not adding an extra dependency just to include the styles.

Reply:
I believe that using the import syntax I suggested it should only pull that single file into the bundle, but that should be tested.  If that's not high-priority we can duplicate for now.